scripts/build-packages-dist.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -u -e -o pipefail
+
+# Go to project directory.
+cd $(dirname ${0})/..
+
+# Either "legacy" (view engine) or "aot" (ivy)
+compileMode=${1:-"legacy"}
+
+# Path to the bazel binary. By default uses "bazel" from the node modules
+# but developers can overwrite the binary though an environment variable.
+bazelBin=${BAZEL_BIN_PATH:-$(yarn bin bazel)}
```

Comment:
Unfortunately we can't. Specifically because we don't want to run `bazel` through yarn/NodeJS as it can cause memory allocation issues in docker containers (i.e. CircleCI). 

I've expanded the comment and renamed the variable to just Bazel to avoid the confusion with the `bazel-bin` folder.